### ISSUE TITLE
Error with steam

### ISSUE THREAD

**Issue by cBournhonesque** [aecsocket/aeronet#67]
When I try to start steam on Mac m2, I get an error like this:
https://github.com/ValveSoftware/GameNetworkingSockets/issues/359

which causes the steam server to immediately close

**Comment by cBournhonesque** [comment]
More info: https://github.com/cBournhonesque/lightyear/issues/1260#issuecomment-3412738212

**Comment by ironpeak** [comment]
I'm getting errors running the steam_server example on main as well, looking into the code I've found that the Steam error is happening here for me:

https://github.com/aecsocket/aeronet/blob/main/crates/aeronet_steam/src/server.rs#L169

~~~rust
    blocking::unblock(move || {
        let result = match target {
            ListenTarget::Addr(addr) => sockets.create_listen_socket_ip(addr, config.to_options()),
            ListenTarget::Peer { virtual_port } => {
                sockets.create_listen_socket_p2p(virtual_port, config.to_options())
            }
        };
        _ = tx_next.send(result.map_err(|_| SessionError::Steam));
    })
    .detach();
~~~

When I print out the error from Steam it's `InvalidHandle`.

When I do the same in the previous version of aeronet (0.16.1) it works.

I've dug a bit through the diff between version but I've not been able to identify why it stopped working.

**Comment by ironpeak** [comment]
I noticed that the SessionConfig.to_options has changed:

https://github.com/aecsocket/aeronet/blob/main/crates/aeronet_steam/src/config.rs#L63

The steam_server example stops crashing when I use an empty vector (old behavior) instead of the new behavior.

Going to try it out tomorrow in my project and see if solves it.

**Comment by ironpeak** [comment]
Version 0.17.2 solves the issue 🚀
